src/plugins/console/public/plugin.ts

```diff
@@ -62,7 +62,7 @@ export class ConsoleUIPlugin implements Plugin<void, void, AppSetupUIPluginDepen
         defaultMessage: 'Console',
       }),
       enableRouting: false,
-      mount: async ({ element }) => {
+      mount: async ({ element, dataSourceId }) => {
         const [core] = await getStartServices();
 
```

Comment:
Should we follow the format as other plugins here? 
```
mount: async (params: AppMountParameters) => {
        const { element, dataSourceId } = params;
```

Reply:
I'd rather keep it, reducing 1 more line of code~